src/main.js

```diff
@@ -1,32 +1,32 @@
-'use strict';
-
-const field = document.getElementById('field');
-const ball = document.getElementById('ball');
-const minTopPosition = 0;
-const minLeftPosition = 0;
-const maxTopPosition = field.clientHeight - ball.clientHeight;
-const maxLeftPosition = field.clientWidth - ball.clientWidth;
-
-field.onclick = function() {
-  const checkedEvent = field.event || event;
-  let leftPosition = checkedEvent.clientX - field.offsetLeft
-    - field.clientLeft - (ball.clientWidth / 2);
-  let topPosition = checkedEvent.clientY - field.offsetTop
-    - field.clientTop - (ball.clientHeight / 2);
-
-  if (leftPosition < minLeftPosition) {
-    leftPosition = minLeftPosition;
-  }
-  if (leftPosition > maxLeftPosition) {
-    leftPosition = maxLeftPosition;
-  }
-  ball.style.left = leftPosition + 'px';
-
-  if (topPosition < minTopPosition) {
-    topPosition = minTopPosition;
-  }
-  if (topPosition > maxTopPosition) {
-    topPosition = maxTopPosition;
-  }
-  ball.style.top = topPosition + 'px';
-};
+'use strict';
+
+const field = document.getElementById('field');

```

Comment:
optional: you global scope if polluted with all these global variables. consider using IFFY https://codeburst.io/javascript-what-the-heck-is-an-immediately-invoked-function-expression-a0ed32b66c18